tests/integration_tests/performance/test_snapshot_ab.py

```diff
@@ -159,23 +167,54 @@ def test_restore_latency(microvm_factory, rootfs, guest_kernel_linux_4_14, test_
     a_revision = os.environ.get("PERFORMANCE_TEST_REVISION_A")
     b_revision = os.environ.get("PERFORMANCE_TEST_REVISION_B")
 
-    ah = test_setup.sample_latency(
-        microvm_factory,
-        guest_kernel_linux_4_14,
-        rootfs,
-        FC_WORKSPACE_DIR / "build" / a_revision,
-    )
-    be = test_setup.sample_latency(
-        microvm_factory,
-        guest_kernel_linux_4_14,
-        rootfs,
-        FC_WORKSPACE_DIR / "build" / b_revision,
+    metrics.set_dimensions(
+        {
+            "instance": global_props.instance,
+            "cpu_model": global_props.cpu_model,
+            "host_kernel": "linux-" + global_props.host_linux_version,
+            "guest_kernel": "linux-4.14",
```

Comment:
can follow the same approach as here: https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/performance/conftest.py#L88-L90

Reply:
Sure! As for `env_id`, we don't need it anymore, as it was only used to index into the baselines dict, and never made it to the metrics, I think.

Reply:
Sync offline, we now instead of using env_id properly submit the microvm configuration as dimensions